kubemarine/core/utils.py

```diff
@@ -386,7 +386,7 @@ def collect_info_all_control_plane(self):
         This method is used to transfer backup logs from the main control-plane to the new control-plane.
         """
 
-        node = self.cluster.nodes['control-plane'].get_initial_nodes().get_first_member(provide_node_configs=True)
+        node = self.cluster.nodes['control-plane'].get_final_nodes().get_initial_nodes().get_first_member(provide_node_configs=True)
```

Comment:
This method is invoked only during add_node procedure, so change is not necessary.

Reply:
fixed too